Makefile

```diff
@@ -40,7 +40,7 @@ build-client-linux:
 
 ## Builds CLI to the executable file huskyci-client
 build-cli:
-	cd cli && $(GO) build -o "$(HUSKYCICLIENTBIN)" main.go
+	cd cli && $(GO) build -o "$(HUSKYCICLIENTBIN)" main.go && mv "$(HUSKYCICLIENTBIN)" ../
```

Comment:
Suggestion:
`$(GO) build -o "$(HUSKYCICLIENTBIN)" cli/main.go`

Reply:
Great! Thanks for this tip. Did it here 9d9b5e7.